src/main/resources/config/liquibase/changelog/20230906184000_changelog.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
+    <changeSet author="erik" id="20230812135300">
+        <comment>
+            Create the table to store multiple paths with a file submission
+            instead of only one and copy the old paths into the new table.
+        </comment>
+        <createTable tableName="file_submission_paths">
+            <column name="submission_id" type="bigint">
+                <constraints nullable="false" />
```

Comment:
Should this also have a foreign key constraint pointing to the `id` column of the `submission` table?